src/scales/scale.linearbase.js

```diff
@@ -23,10 +25,15 @@ function generateTicks(generationOptions, dataRange) {
 	var precision = generationOptions.precision;
 	var spacing, factor, niceMin, niceMax, numSpaces;
 
+	// sanitize dataRangee to MAX_PRECISION
```

Comment:
extra "e" in "dateRange"

Reply:
will fix this tomorrow with fresh eyes 😪